pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -61,6 +62,8 @@
     private static final LongAdder bytesReceived = new LongAdder();
     private static final DecimalFormat dec = new DecimalFormat("0.000");
 
+    private static Recorder recorder = new Recorder(TimeUnit.DAYS.toMillis(90), 5);
```

Comment:
Do you have practical reason about `TimeUnit.DAYS.toMillis(90)` ?
In PerformanceProducer, `TimeUnit.SECONDS.toMillis(120000)` is used.

Reply:
I haven't have practical reason, I thought that duration is a bit short, about 33 hours, what's your suggested duration?